tasks/89_symmertic-difference/symmertic-difference.js

```diff
@@ -0,0 +1,15 @@
+export function sym(...args) {
+  return args
+      .reduce((result, curr) => {
+        curr = [...new Set(curr)];
+        curr.forEach((item) => {
+          if (!result.includes(item)) {
```

Comment:
you can simplify it a lot if instead of array as result you will use Set, so you will need just call `has` method and `add` to Set or if Set already has this value call delete `method`